### ISSUE TITLE
Doesn't increase cursors when we use not English (Select word or next occurrence of selection)

### ISSUE THREAD

**Issue by tadashi-aikawa** [timhor/obsidian-editor-shortcuts#37]
Thank you for the great plugin! Especially, `Select word or next occurrence of selection` is very exciting for me :)

I found an unexpected behavior when I used `Select word or next occurrence of selection` other than English.

## Reproduction procedure

1. Input words other than English (Ex: Japanese, Chinese)
2. Exec `Select word or next occurrence of selection`

![2022-06-07](https://user-images.githubusercontent.com/9500018/172287529-e5cfe91c-7106-4558-b683-4b19716e5a35.gif)

```
ああ ああ ああ
```

It succeeded in selecting the current word but didn't increase cursors.

## Expected

![2022-06-07](https://user-images.githubusercontent.com/9500018/172287980-6a568761-6848-46c5-b0ed-c1e8170d086a.gif)

## Note

I read your code and found it to satisfy my requirements by changing it as follows.

https://github.com/timhor/obsidian-editor-shortcuts/blob/master/src/actions.ts#L175

```diff
    const nextMatch = findNextMatchPosition({
      editor,
      latestMatchPos,
      searchText,
-     searchWithinWords: isManualSelection,
+     searchWithinWords: true,
      documentContent: editor.getValue(),
    });
```

But I believe it is contrary to your intent. It would be great to solve my problems if you know better solutions :)

Thank you!!

**Comment by timhor** [comment]
Thanks for reporting! I'll have a fix for this in the next release.

A temporary workaround (apart from changing the code) would be to make the selection manually using your keyboard or mouse _before_ running the command.

**Comment by timhor** [comment]
This is now fixed in [v1.10.0](https://github.com/timhor/obsidian-editor-shortcuts/releases/tag/1.10.0)
